jpf-symbc/witness_template/witness_template_minimal.txt

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Same as the "witness_template.txt", need to exist in the resources folder of the project.

Reply:
I moved witness template to src/main/resources, thus I'm closing this review. Thank you.